modules/installation-configuration-parameters.adoc

```diff
@@ -424,7 +424,7 @@ Configures the IPv4 join subnet that is used internally by `ovn-kubernetes`. Thi
 Optional installation configuration parameters are described in the following table:
 
 .Optional parameters
-[cols=".^2l,.^3a,.^3a",options="header"]
+[cols=".^22,.^3a,.^3a",options="header"]
```

Comment:
This was an `l`, not a `1` - it's providing the "literal" format operator to values in this column ([more info](https://docs.asciidoctor.org/asciidoc/latest/tables/format-column-content/)).
```suggestion
[cols=".^2l,.^3a,.^3a",options="header"]
```